### PR TITLE
Document the required cpp-ethereum commit hash

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
Need to fix the build failure. Alternatively we could keep the commit hash in the documentation and keep a comment in `tests.sh` saying whenever it changes there the documentation must be updated too.

File: docs/contributing.rst [hunk 1]
```diff
@@ -93,6 +93,12 @@ Alternatively, there is a testing script at ``scripts/test.sh`` which executes a
 
 Travis CI even runs some additional tests (including ``solc-js`` and testing third party Solidity frameworks) that require compiling the Emscripten target.
 
+.. note ::
+
+While any version of ``cpp-ethereum`` should be usable, this cannot be guaranteed, and it is suggested to use the same version as used by the Solidity continuous integration tests are doing.
```
Comment (chriseth) [inline docs/contributing.rst]: `as used` ... `are doing`

Comment (ekpyron) [reply docs/contributing.rst]: Should we explicitly mention that certain commit ranges of ``cpp-ethereum`` are known to fail due to bugs?

Comment (axic) [reply docs/contributing.rst]: I wouldn't go as far as that, but at least mention the hash we are using, which is supposed to work.

Comment (ekpyron) [inline docs/contributing.rst]: This block needs indentation. Because it's missing the docs build is failing.

**Review by ekpyron — DISMISSED** [review]

**Comment by axic** [comment]
@chriseth @ekpyron updated!

**Comment by chriseth** [comment]
Rebased and corrected grammar.

**Review by chriseth — APPROVED** [review]

**Pull request merged**